design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -143,6 +143,67 @@ The proposed API changes for VM objects can be seen in
 [the VM API examples section](#vms), while the proposed API changes for the VMI
 object can be seen in [the VMI API examples section](#vmis).
 
+### Pod interface naming
+The VMs’ pod interfaces names are ordinal based (`net1`, `net2`, …, `netX`),
+derived from their order in the VMI spec.
+They are requested from Multus by specifying then in virt-launcher pod 
+`k8s.v1.cni.cncf.io/networks` annotation (which is created by virt-controller).
+
+Given a VM with three secondary interfaces:
+```yaml
+spec:
+  networks:
+  - name: blue-network
+    multus: ...
+  - name: red-network
+    multus: ...
+  - name: green-network
+    multus: ...
+```
+
+The pod's Multus networks annotation will look like so:
+```json
+"k8s.v1.cni.cncf.io/networks": [
+  {"interface": "net1", ...},
+  {"interface": "net2",  ...},  
+  {"interface": "net3",  ...}
+]
+```
+
+In the scenario where "red-network" interface is unplugged, the annotation will change  as follows:
+```json
+"k8s.v1.cni.cncf.io/networks": [
+  {"interface": "net1", ...},
+  {"interface": "net3",  ...}
+]
+```
+Now it's impossible to associate between all VMI spec and pod interfaces.
+
+Thus, the name of the `virt-launcher` pod network interfaces must be generated 
+without relying on the interface order in the spec, allowing the unplug feature.
+
+The pod interface name will be derived from the `kubevirt-spec-iface-name`; 
+we'll simply compute a Hash of the interface name
+(which is guaranteed to be unique within each VMI), and ensure all generated
+names for the pod's networking infrastructure are accepted by the kernel.
+Refer to the following list for examples of names on pod networking infra:
+- VM interface name: `iface1`
+- pod interface name: `7e0055a6880`
```

Comment:
this is not aligned w/ the implementation proposed in https://github.com/kubevirt/kubevirt/pull/9268#discussion_r1196207065